### PR TITLE
update rustfmt

### PR DESCRIPTION
**Pull request by saolsen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ requirements:
 - Ruby: 2.4+
   - Bundler 2.1.4+
 - Rust: 1.46+
+- Go: 1.12+
```
Comment (samscott89) [inline README.md:68]: How did you/we determine this? Do we test on this version?

**Review by samscott89 — APPROVED** [review]
Not sure I agree with rustfmt on these changes, but oh well.

File: README.md (hunk 1 above)
Comment (saolsen) [reply README.md:68]: This is the first version that go modules were officially released.

**Review by gj — DISMISSED** [review]
will make badge

**Pull request merged**